### PR TITLE
Added 4-bit Synchronous Counters Simulation

### PR DESCRIPTION
**Pull request by AymanNasser**
# ref #116, #92 
## Changes Done:
- [x] Added Synchronous Up Counter Simulation
- [x] Added Synchronous Down Counter Simulation
- [x] Added Synchronous Up/Down Counter Simulation
- [x] Added Synchronous Decade Counter Simulation
- [x] Modified Synchronous Decade image bec. **J-FF0** wasn't connected to high, so it was confusing   

## Preview link:
**https://deploy-preview-345--tender-ardinghelli-ebb79f.netlify.app/docs/counter**

## By submitting this PR, I have verified the following
- [x] Checked to see if a similar PR has already been opened thinking
- [x] Reviewed the contributing guidelines mag
- [x] Sample preview link added (add a link from the checks tab after checks complete)

### CODE REVIEW

**Comment by AymanNasser** [comment]
@YashKumarVerma You can check the PR & I'm waiting for your comment

**Comment by Shivansh2407** [comment]
Also try to implement the other counter as well like the 4 Bit Down Counter, Up/Down Counter etc.

**Comment by YashKumarVerma** [comment]
> @YashKumarVerma You can check the PR & I'm waiting for your comment

Shivansh has given an apt review. Let us know if you need any help.

**Comment by AymanNasser** [comment]
Where can i find the PR template ? bec. I'm new to open source project contribution

**Comment by mahmodHammad** [comment]
Nice work.
Please share a gif or a preview link

**Comment by mahmodHammad** [comment]
You will fin the PR template at the previous PRs
, for simplicity all we need is a brief description, .Gif for the i frame, or preview link

**Comment by AymanNasser** [comment]
**Hey guys, Check the pr after modifications, waiting for your reviews**

**Comment by Shivansh2407** [comment]
Nice work @AymanNasser 👍. Please confirm that this image that you are using is not copyrighted. I mean to say that it is available for reuse.

**Comment by AymanNasser** [comment]
> Nice work @AymanNasser . Please confirm that this image that you are using is not copyrighted. I mean to say that it is available for reuse.

- [x] image doesn't contain any contact info or email address 
- [x] image doesn't contain any water mark
- [x] did google reverse image search & couldn't find any useful data
So i think its safe to use it

File: docs/counter.md [hunk 1]
```diff
@@ -82,22 +82,27 @@ The JA and KA inputs of FF-A are tied to logic 1. So FF-A will work as a toggle
 * **4 bit synchronous up counter:** In the up counter the 4 bit binary sequence starts from 0000 and increments up to 1111, i.e from 0 to 15.
 
 <div style="text-align:center"><img src="../assets/images/4bit_up_counter.png" height="300"/></div>
+
+<iframe width="400%" height="400px" src="https://circuitverse.org/simulator/embed/93827" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen></iframe>
```
Comment (mahmodHammad) [inline docs/counter.md:86]: image
Please fix the width issue.
i think when you tried to change the 400px to 100% you made it 400%. 😄

Comment (AymanNasser) [reply docs/counter.md]: Fixed it to 100% @mahmodHammad

**Comment by mahmodHammad** [comment]
I think we shouldn't worry about the copyrights if the image right now, we will change it any way during DSOC

**Pull request merged**